test/services/bulkstatus.service.test.js

```diff
@@ -81,6 +82,33 @@ describe('Dynamic X-Progress logic', () => {
         expect(response.headers['x-progress']).toEqual('60.00% Done');
       });
   });
+  test('check operationOutcome includes the number of resources when available', async () => {
+    const response = await supertest(server.app)
```

Comment:
This is not quite the approach we want to take with the unit tests. What you had in the previous commit was almost it, we just needed to add different assertions and update the test fixtures a bit.

These unit tests should only be testing the bulkstatus endpoint. The reason we have all those bulkstatus test fixtures in that JSON file is so we don't have to go through the import flow within the unit tests. We can just call existing bulkstatus endpoints with known client IDs directly and control what they return. We don't need to call import and grab the content location.

You have a test fixture for a bulkstatus with ID `COMPLETED_REQUEST_WITH_RESOURCE_COUNT`, and you have complete control over what a GET request to `/4_0_1/bulkstatus/COMPLETED_REQUEST_WITH_RESOURCE_COUNT` should return. Namely, whatever JSON exists for that test fixture in the `testBulkStatus.json` file.

Your  code introduces a new property `successCount` on these bulkStatus objects that represents the number of successfully parsed resources, and `bulkStatus.successCount` is what we use to populate the OperationOutcome text with the count. So the unit test changes to get a proper OperationOutcome should involve updating any fixtures and calls to make sure that `successCount` is properly set on the returned bulkStatus. That way, you're testing whether or not the code properly takes that number and puts it on the OperationOutcome resources.

You'll still need to have a supertest request to `/4_0_1/file/COMPLETED_REQUEST_WITH_RESOURCE_COUNT/OperationOucome.ndjson`, but you should be able to hardcode the clientID since it's the same as the bulkstatus request endpoint. You'll still need to look at newResponse.text and make your assertions there, but you need to assert that the proper `successCount` value lives within the OperationOutcome text. This current assertion is just asserting that string "successfully imported  " is contained somewhere within the entire stringified resource, which doesn't quite hit the case that this task is aimed at.